UI/qt-display.cpp

```diff
@@ -62,6 +70,38 @@ static inline QColor rgba_to_color(uint32_t rgba)
 			       (rgba >> 16) & 0xFF, (rgba >> 24) & 0xFF);
 }
 
+static bool QTToGSWindow(QWindow *window, gs_window &gswindow)
```

Comment:
This looks like new code?

Reply:
Moved see the commit for more context about this "new code", QTToGSWindow is only used once by the UI there is no point keeping it in the wrapper.